packages/platform-sdk/test/integrationTests/customer/customer.test.ts

```diff
@@ -0,0 +1,210 @@
+import { randomUUID } from 'crypto'
+import { apiRoot } from '../test-utils'
+import {
+  _BaseAddress,
+  BaseAddress,
+  CustomerDraft,
+  CustomerGroupResourceIdentifier,
+  StoreResourceIdentifier,
+} from '../../../src'
+import {
+  createCustomerGroup,
+  deleteCustomerGroup,
+} from '../customer-group/customer-group-fixture'
+import {
+  createCustomer,
+  createCustomerDraft,
+  deleteCustomerFromUpdatableObject,
+} from './customer-fixture'
+import { createStore, deleteStore } from '../store/store-fixture'
+
+describe('testing customer API calls', () => {
+  it('should create and delete a customer by ID', async () => {
+    const customerGroup = await createCustomerGroup()
+
+    const customerGroupResourceIdentifier: CustomerGroupResourceIdentifier = {
+      typeId: 'customer-group',
+      id: customerGroup.body.id,
+    }
+
+    const address: BaseAddress[] = [
+      {
+        id: randomUUID(),
+        country: 'DE',
+      },
+    ]
+    const customerDraft: CustomerDraft = {
+      key: randomUUID(),
+      email: 'test-email-customer' + randomUUID(),
+      password: 'test-password-customer' + randomUUID(),
+      customerGroup: customerGroupResourceIdentifier,
+      addresses: address,
+    }
+
+    const responseCreatedCustomer = await apiRoot
+      .customers()
+      .post({ body: customerDraft })
+      .execute()
+
+    expect(responseCreatedCustomer.statusCode).toEqual(201)
+    expect(responseCreatedCustomer.body.customer).not.toBe(null)
+
+    const responseCustomerDeleted = await apiRoot
+      .customers()
+      .withId({ ID: responseCreatedCustomer.body.customer.id })
+      .delete({
+        queryArgs: { version: responseCreatedCustomer.body.customer.version },
+      })
+      .execute()
+
+    expect(responseCustomerDeleted.statusCode).toEqual(200)
+
+    await deleteCustomerGroup(customerGroup)
+  })
+
+  it('should update a customer by Id', async () => {
+    const customerGroup = await createCustomerGroup()
```

Comment:
Leaving these methods here to show the customer step-by-step how to create/delete the final object